README.md

```diff
@@ -1,8 +1,14 @@
 # Repository for distributing (some) stan-dev R packages
 
-A place for publishing new versions of (some) `stan-dev` R packages before they reach CRAN and for `stan-dev` R packages and versions where releasing on CRAN is not a (current) goal.
+A place for publishing new versions of (some) `stan-dev` R packages before they reach CRAN and for `stan-dev` R packages and versions where releasing on CRAN is not a (current) goal. As of 2021-03-16 this is most relevant for `rstan`, where the CRAN version is unfortunately several releases behind and pushing a new version to CRAN has been difficult.
 
-**Instructions for publishing or updating on this repository**
+To install latest `rstan` or other packages from the repo, add `https://mc-stan.org/r-packages/` to your repository list, e.g.:
```

Comment:
Before going into how to install, can we just add a note on which packages are currently hosted here:

- cmdstanr
- posterior
- rstan
- rstanarm (survival branch - https://arxiv.org/pdf/2002.09633.pdf)

Reply:
Actually I can add that separately, no sense of bugging you with this.